ha/lock_test.go

```diff
@@ -0,0 +1,334 @@
+//go:build !race
+// +build !race
+
+package ha
+
+import (
+	"context"
+	"errors"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/nomad/helper/testlog"
+	"github.com/shoenig/test/must"
+)
+
+var lease = 10 * time.Millisecond
```

Comment:
```suggestion
const defaultLockLease = 10 * time.Millisecond
```

This value is still global in the package, so a generic name could clash with another variable and cause some unexpected behaviours.